web/pgadmin/static/js/components/Buttons.jsx

```diff
@@ -37,18 +37,19 @@ const StyledButton = styled(Button)(({theme}) => ({
     }
   },
   '&.Buttons-defaultButton': {
-    backgroundColor: theme.palette.default.main,
-    color: theme.palette.default.contrastText,
-    border: '1px solid '+theme.palette.default.borderColor,
+    // Use the color prop to determine the background color and text color.
+    backgroundColor: color === 'error' ? theme.palette.error.main : theme.palette.default.main,
```

Comment:
Don't use ternary operator here. if we want some other color then it will jump to default.